### ISSUE TITLE
SerenityManagedException is thrown when getAttribute() returns a boolean value but compared as string

### ISSUE THREAD

**Issue by amodmahajan2016** [serenity-bdd/serenity-core#2021]
Step to reproduce:-
1. Call getAttribute("attribute name whose value is boolean i.e. disabled etc")
2. Do not parse the value and call equals or contains method

Sample code:-
Assert.assertTrue(element.getAttribute("disabled").equals(true) );

I was getting SerenityManagedException and was not able to understand what is actual issue. 
May be something more details should be given with above exception.

**Comment by wakaleo** [comment]
The `getAttribute()` method always returns a String (thats how the WebDriver API works) so it will be your responsibility to convert it to a boolean type if you want to compare it with a boolean. SerenityManagedException just wraps other exceptions, so you need to look further in the stack trace to see the actual issue. 

For the assertion itself, you could try something like this (using AssertJ)
```
assertThat(element.getAttribute("disabled")).isEqualToIgnoringCase("true")
```
or (using Hamcrest):
```
assertThat(element.getAttribute("disabled"), equalToIgnoringCase("true"))
```

**Comment by amodmahajan2016** [comment]
Yes you are correct. I should have converted it to boolean or string equivalent of boolean value as above example given by you. This I figured it out later. In stack trace I got the line but not able to get quickly what went wrong. May be we should get some hints. Whenever I get SereniryMangedException, I could not get the actual issue quickly. It may be lack of my knowledge in Serenity BDD. Thanks.
